Silicon/NXP/iMX7Pkg/iMX7CommonDsc.inc

```diff
@@ -838,22 +839,57 @@
   }
 !endif
 
+  #
   # Applications
-  MdeModulePkg/Application/HelloWorld/HelloWorld.inf
-  #Platform/Microsoft/Application/StorageTest/StorageTest.inf
-#  Silicon/NXP/iMX7Pkg/Application/ImxClkPwr/ImxClkPwr.inf
-
+  #
+  # For each application, you can include stdlib libraries from StdLib/StdLib.inc.
+  # This stdlib implementation is only intended for UEFI Applications. It
+  # modifies the compiler's CC_FLAGS to ignore all other stdlib implementations
+  # which could break firmware builds if the StdLib.inc is included haphazardly.
+  # This stdlib is located in tianocore/edk2-libc and must be in your
+  # PACKAGES_PATH environment variable during application build.
+  #
 !if $(CONFIG_EDK_TESTS) == TRUE
-  Platform/Microsoft/Application/StorageTest/StorageTest.inf
-  Platform/Microsoft/Application/AuthVarTest/AuthVarTest.inf
+  Platform/Microsoft/Application/StorageTest/StorageTest.inf {

```

Comment:
This is really ugly.  Consider looking at the changes in the Project Mu repos for AuthVar and Storage tests.  It removes the dependency on stdlib entirely.

Reply:
https://github.com/ms-iot/MU_PLATFORM_NXP/commit/b6d2130d3ddfb654cd823276dd8b158126c21dc7

Will make similar changes so we can remove the stdlib dependency from the project altogether.